### DISCUSSION TITLE
Connect 2 or more Directus for staging

### DISCUSSION THREAD

**Discussion by casualmatt** [directus/directus#11312] [Feature Requests]
Hello, 

we always have 3 copy of the "same" Directus install, one is the one where we actually develop on top, the 2° one is the testing and the 3° is the production. 
But this mean that we need to copy and paste schema everything from one install to the other, i could be so easy if we can connect the install, and have a way to apply a schema to others Directus with out using the [CLI](https://docs.directus.io/reference/cli/#migrate-schema-to-a-different-environment) then download the yaml and re upload it to the next server.

**Comment by rijkvanzanten** [comment]
@casualmatt Would be able to provide an example of how you'd imagine to use this? Would it be something like

```
directus schema migrate-to https://another.example.com/
```

?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
